chevah/brink
# [#4587] Add support for publishing the trial version.

Scope
=====

This is a small update on the release publish code to also push the trial versions of a product.


Changes
=======

The heavy lifting is done by each repo itself, this only copies the trial to trial and not-trial to normal release.


I have updated paver.sh .

The release notes for 0.67 was not updated, but is fine. I don't want to spend to much time with it.

How to try and test the changes
===============================

reviewers: @bgola 

Check that changes make sense.

This version is already used in the server at https://chevah.com/buildbot/builders/server-gk-release-staging/builds/8/steps/publish/logs/stdio
